medusa/search/manual.py

```diff
@@ -177,12 +172,13 @@ def collect_episodes_from_search_thread(show):
     return episodes
 
 
-def get_provider_cache_results(indexer, show_all_results=None, perform_search=None, **search_show):  # pylint: disable=too-many-locals,unused-argument
+def get_provider_cache_results(indexer, show_all_results=None, perform_search=None, show=None,
+                               season=None, episode=None, manual_search_type=None, **search_show):
     """Check all provider cache tables for search results."""
-    show = search_show.get('show')
-    season = search_show.get('season')
-    episode = search_show.get('episode')
-    manual_search_type = search_show.get('manual_search_type')
+    show = show
```

Comment:
Not sure if inside the method it will (would have to look at the code) but calling the method itself would not cause a traceback because default values are provided.